### PR TITLE
Remove vendoring of USWDS with jekyll-assets plugin and clean up unused Jekyll pages search

### PR DESCRIPTION
**Pull request by apburnes**
Fixes issue(s) #3311 and partially #3286

[:sunglasses: PREVIEW](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.app.cloud.gov/preview/18f/18f.gsa.gov/apb/remove-uswds-vendoring)

Changes proposed in this pull request:
- Migrate USWDS assets from being vendored and stored in git by using the jekyll assetss plugin
- Remove unused jekyll pages search and jekyll pages api from site

### CODE REVIEW

File: _sass/_components/social-links.scss [hunk 1]
```diff
@@ -1,9 +1,8 @@
 .usa-social-link--github {
-    background-image: url('../img/social-icons/svg/github-dark.svg'), linear-gradient(transparent, transparent);
+    background-image: url('../assets/img/social-icons/svg/github-dark.svg'), linear-gradient(transparent, transparent);
     background-repeat: no-repeat;
   }
 .usa-social-link--linkedin {
-    background-image: url('../img/social-icons/svg/linkedin.svg'), linear-gradient(transparent, transparent);
+    background-image: url('../assets/img/social-icons/svg/linkedin.svg'), linear-gradient(transparent, transparent);
```
Comment (apburnes) [inline _sass/_components/social-links.scss:6]:
```suggestion
  background-image: url('../assets/img/social-icons/svg/linkedin.svg'), linear-gradient(transparent, transparent);
```

File: _sass/_components/social-links.scss [hunk 2]
```diff
@@ -1,9 +1,8 @@
 .usa-social-link--github {
-    background-image: url('../img/social-icons/svg/github-dark.svg'), linear-gradient(transparent, transparent);
+    background-image: url('../assets/img/social-icons/svg/github-dark.svg'), linear-gradient(transparent, transparent);
```
Comment (apburnes) [inline _sass/_components/social-links.scss:2]:
```suggestion
  background-image: url('../assets/img/social-icons/svg/github-dark.svg'), linear-gradient(transparent, transparent);
```

File: _sass/_components/nav-accordion.scss [hunk 3]
```diff
@@ -32,13 +32,13 @@
...
-        background-image: url('../img/global/minus-alt.svg');
+        background-image: url('../assets/img/global/minus-alt.png');
+        background-image: url('../assets/img/global/minus-alt.svg');
       }
 
       &[aria-expanded=false] {
-        background-image: url('../img/global/plus-alt.png');
-        background-image: url('../img/global/plus-alt.svg');
+        background-image: url('../assets/img/global/plus-alt.png');
+        background-image: url('../assets/img/global/plus-alt.svg');
```
Comment (apburnes) [inline _sass/_components/nav-accordion.scss:41]:
```suggestion
```

File: _sass/_components/nav-accordion.scss [hunk 4]
```diff
@@ -32,13 +32,13 @@
       }
 
       &[aria-expanded=true] {
-        background-image: url('../img/global/minus-alt.png');
-        background-image: url('../img/global/minus-alt.svg');
+        background-image: url('../assets/img/global/minus-alt.png');
+        background-image: url('../assets/img/global/minus-alt.svg');
```
Comment (apburnes) [inline _sass/_components/nav-accordion.scss:36]:
```suggestion
```

File: _sass/_components/layout.scss [hunk 5]
```diff
@@ -171,8 +171,8 @@
   }
 
   .usa-search.usa-search-small [type="submit"] {
-    background-image: url("../img/global/search-dark.png");
-    background-image: url("../img/global/search-dark.svg");
+    background-image: url("../assets/img/global/search-dark.png");
+    background-image: url("../assets/img/global/search-dark.svg");
```
Comment (apburnes) [inline _sass/_components/layout.scss:175]:
```suggestion
```

Comment (apburnes) [inline _sass/_components/layout.scss:175]:
```suggestion
    background-image: url('../assets/img/global/search-dark.svg');
```

File: _sass/_components/layout.scss [hunk 6]
```diff
@@ -171,8 +171,8 @@
   }
 
   .usa-search.usa-search-small [type="submit"] {
-    background-image: url("../img/global/search-dark.png");
-    background-image: url("../img/global/search-dark.svg");
+    background-image: url("../assets/img/global/search-dark.png");
```
Comment (apburnes) [inline _sass/_components/layout.scss:174]:
```suggestion
    background-image: url('../assets/img/global/search-dark.png');
```

File: _sass/_components/footer.scss [hunk 7]
```diff
@@ -55,13 +55,13 @@ footer[role=contentinfo] {
...
-      background-image: url("../img/global/plus-alt.svg");
+      background-image: url("../assets/img/global/plus-alt.png");
+      background-image: url("../assets/img/global/plus-alt.svg");
     }
 
     [aria-expanded=true] {
-      background-image: url("../img/global/minus-alt.png");
-      background-image: url("../img/global/minus-alt.svg");
+      background-image: url("../assets/img/global/minus-alt.png");
+      background-image: url("../assets/img/global/minus-alt.svg");
```
Comment (apburnes) [inline _sass/_components/footer.scss:64]:
```suggestion
```

File: _sass/_components/footer.scss [hunk 8]
```diff
@@ -55,13 +55,13 @@ footer[role=contentinfo] {
...
-      background-image: url("../img/global/plus-alt.png");
-      background-image: url("../img/global/plus-alt.svg");
+      background-image: url("../assets/img/global/plus-alt.png");
+      background-image: url("../assets/img/global/plus-alt.svg");
     }
 
     [aria-expanded=true] {
-      background-image: url("../img/global/minus-alt.png");
-      background-image: url("../img/global/minus-alt.svg");
+      background-image: url("../assets/img/global/minus-alt.png");
```
Comment (apburnes) [inline _sass/_components/footer.scss:63]:
```suggestion
      background-image: url('../assets/img/global/minus-alt.png');
```

File: _sass/_components/footer.scss [hunk 9]
```diff
@@ -55,13 +55,13 @@ footer[role=contentinfo] {
     }
 
     [aria-expanded=false] {
-      background-image: url("../img/global/plus-alt.png");
-      background-image: url("../img/global/plus-alt.svg");
+      background-image: url("../assets/img/global/plus-alt.png");
+      background-image: url("../assets/img/global/plus-alt.svg");
```
Comment (apburnes) [inline _sass/_components/footer.scss:59]:
```suggestion
```

File: _sass/_components/footer.scss [hunk 10]
```diff
@@ -55,13 +55,13 @@ footer[role=contentinfo] {
     }
 
     [aria-expanded=false] {
-      background-image: url("../img/global/plus-alt.png");
-      background-image: url("../img/global/plus-alt.svg");
+      background-image: url("../assets/img/global/plus-alt.png");
```
Comment (apburnes) [inline _sass/_components/footer.scss:58]:
```suggestion
      background-image: url('../assets/img/global/plus-alt.png');
```

File: _sass/_components/buttons.scss [hunk 11]
```diff
@@ -56,8 +56,8 @@ button,
   }
 
   &:hover::#{$direction} {
-    background-image: url('../img/#{$icon}-hover.png');
-    background-image: url('../img/#{$icon}-hover.svg');
+    background-image: url('../assets/img/#{$icon}-hover.png');
+    background-image: url('../assets/img/#{$icon}-hover.svg');
```
Comment (apburnes) [inline _sass/_components/buttons.scss:60]:
```suggestion
```

File: _sass/_components/buttons.scss [hunk 12]
```diff
@@ -39,8 +39,8 @@ button,
 
 @mixin display-icon($icon, $direction, $size) {
   &:#{$direction} {
-    background-image: url('../img/#{$icon}.png');
-    background-image: url('../img/#{$icon}.svg');
+    background-image: url('../assets/img/#{$icon}.png');
+    background-image: url('../assets/img/#{$icon}.svg');
```
Comment (apburnes) [inline _sass/_components/buttons.scss:43]:
```suggestion
```

File: _sass/_components/accordion.scss [hunk 13]
```diff
@@ -1,11 +1,11 @@
...
-    background-image: url('../img/arrow-down.svg');
+    background-image: url('../assets/img/arrow-down.png');
+    background-image: url('../assets/img/arrow-down.svg');
   }
 
   &[aria-expanded=true] {
-    background-image: url('../img/arrow-up.png');
-    background-image: url('../img/arrow-up.svg');
+    background-image: url('../assets/img/arrow-up.png');
+    background-image: url('../assets/img/arrow-up.svg');
```
Comment (apburnes) [inline _sass/_components/accordion.scss:9]:
```suggestion
```

File: _sass/_components/accordion.scss [hunk 14]
```diff
@@ -1,11 +1,11 @@
 .usa-accordion-button-chevron {
   &[aria-expanded=false] {
-    background-image: url('../img/arrow-down.png');
-    background-image: url('../img/arrow-down.svg');
+    background-image: url('../assets/img/arrow-down.png');
+    background-image: url('../assets/img/arrow-down.svg');
```
Comment (apburnes) [inline _sass/_components/accordion.scss:4]:
```suggestion
```

**Comment by apburnes** [comment]
@Dahianna nothing should change on the site (I couldn't find anything) and we are just updating the way we build USWDS styles.  Just a double check between the preview and production sites to make sure everything looks the same.

**Review by Dahianna — APPROVED** [review]

**Pull request merged**